src/constants/networks.ts

```diff
@@ -378,4 +345,14 @@ export const NETWORKS_INFO_MAP: { [id in ChainId]: NetworkInfo } = {
   [ChainId.OPTIMISM]: OptimismNetworkInfo,
 }
 
-export const SUPPORTED_NETWORKS = [...SHOW_NETWORKS, ALL_CHAIN_ID].filter((e) => e != ChainId.AURORA) as ChainIdType[]
+// all network info
+export const NETWORKS_INFO_LIST: NetworkInfo[] = Object.values(NETWORKS_INFO_MAP)
+
+const NETWORK_NOT_SUPPORT = [ChainId.ROPSTEN, ChainId.RINKEBY, ChainId.AURORA]
+
+const FILTER_NETWORK = Object.keys(NETWORKS_INFO_MAP)
+  .map(Number)
+  .filter((e) => !NETWORK_NOT_SUPPORT.includes(e))
+
+// for fetch data, display
+export const SUPPORTED_NETWORKS = [ALL_CHAIN_ID, ...FILTER_NETWORK] as ChainIdType[]
```

Comment:
This is still ambigous I think. I propose we gonna do like this:
 - `SUPPORTED_NETWORKS` will contains all networks's chain id that we support (including testnet and aurora). This gonna equal current `NETWORKS_INFO_LIST`
 - `ALL_SUPPORTED_NETWORKS` will like above, with additional `ALL_CHAIN_ID`
 - `CLASSIC_SUPPORTED_NETWORKS` will contains all networks's chain id that we support for classic
 - `ELASTIC_SUPPORTED_NETWORKS` will equal current `FILTER_NETWORK`
 
`NETWORK_NOT_SUPPORT` is very `tối nghĩa` for me. I suggest not to use it.

What do you think? @nguyenhoaidanh

Reply:
`ALL_SUPPORTED_NETWORKS` may unused in this case we already have `CLASSIC_SUPPORTED_NETWORKS` and `ELASTIC_SUPPORTED_NETWORKS`

Reply:
agree all.

NETWORK_NOT_SUPPORT is very tối nghĩa for me. I suggest not to use it. => currently, it is local on that file, we can rename it if you want